taichi/codegen/codegen_llvm.cpp

```diff
@@ -1075,8 +1079,26 @@ void CodeGenLLVM::visit(GlobalStoreStmt *stmt) {
 void CodeGenLLVM::visit(GlobalLoadStmt *stmt) {
   int width = stmt->width();
   TI_ASSERT(width == 1);
-  llvm_val[stmt] = builder->CreateLoad(tlctx->get_data_type(stmt->ret_type),
-                                       llvm_val[stmt->ptr]);
+  if (auto cit = stmt->ret_type->cast<CustomIntType>()) {
```

Comment:
Here I guess testing `stmt->ptr->ret_type->cast<PointerType>()->is_bit_pointer()` is better here?

Reply:
Yes, you’re right 👍

Reply:
> Here I guess testing `stmt->ptr->ret_type->cast<PointerType>()->is_bit_pointer()` is better here?

I tried to test `stmt->ptr->ret_type->cast<PointerType>()->is_bit_pointer()`, but it seems always False. Maybe we need to have a look at when do we set ret_type bit-level.

Reply:
OK, sounds good. I'll leave this task to you :-)